changelog.md

```diff
@@ -8,6 +8,7 @@
 - Creating the Top Cut for a tournament is a separate `mc!topcut` command that can be called on a tournament after `mc!finish` is used. This allows taking a break between the end of Swiss and the start of Top Cut.
 - New `mc!tie` command changes all open matches in the current round to a tie. This is useful when a number of matches are outstanding at the end of a round or tournament.
 - A custom timer length can now be specified for `mc!round` and `mc!start` (for the first round).
```

Comment:
Not updating this is my mistake in another branch but it's no longer true that you specify a round length for start